src/main/java/com/teamabnormals/abnormals_core/core/util/registry/ItemSubRegistryHelper.java

```diff
@@ -63,6 +63,20 @@ public ItemSubRegistryHelper(RegistryHelper parent) {
 		return item;
 	}
 
+	/**
+	 * Creates and registers a compat {@link Item}
+	 *
+	 * @param name       - The name for the item
+	 * @param properties - The item's properties
+	 * @param group      - The {@link ItemGroup} for the {@link Item}
+	 * @param modIds     - The mod ids of the mods this block is compatible for
+	 * @return A {@link RegistryObject} containing the {@link Item}
+	 */
+	public RegistryObject<Item> createCompatItem(String name, Item.Properties properties, ItemGroup group, String... modIds) {
+		RegistryObject<Item> item = this.deferredRegister.register(name, () -> new Item(properties.group(areModsLoaded(modIds) ? group : null)));
```

Comment:
Don't have `item` be a local variable, just return `this.deferredRegister.register(name, () -> new Item(properties.group(areModsLoaded(modIds) ? group : null)))`.

Reply:
The original `createCompatItem` method has `item` as a local variable and then returns `item`. In the interest of keeping things the same across the overloads, would you like me to change the original one as well?

Reply:
Change the original, yeah.